### PR TITLE
Clarifications and updates: NLNet, DIAL, UNICEF

### PR DESCRIPTION
**Pull request by brainwane**

### CODE REVIEW

**Comment by brainwane** [comment]
Ready for review & merge!

File: funders.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ https://unicefinnovationfund.org/
 
 "The Innovation Fund allows UNICEF to quickly assess, fund and grow open-source solutions that can improve children’s lives. Financial and technological support is available for companies that are using technology in innovative ways to improve the world."
 
-[Eligibility criteria include](https://unicefinnovationfund.org/#/submit):  You must be registered as a private company, and your company/institution must be registered in a UNICEF programme country.
+[Eligibility criteria include](https://unicefinnovationfund.org/#/submit):  You must be registered as a private company, and your company/institution must be registered in a UNICEF programme country (note: the United States is not a UNICEF programme country).
 
 Amount: approximately up to US$90,000.
```
Comment (xmunoz) [inline funders.md]: It's 100K: https://www.unicefinnovationfund.org/funding-support

**Review by xmunoz — APPROVED** [review]
Feel free to merge after you update the funding amount for UNICEF.

**Comment by xmunoz** [comment]
Also, not sure if you want to add it to this PR, but Comcast Innovation Fund is closed for this year:
Screenshot from 2020-10-03 10-07-07

**Comment by brainwane** [comment]
Thanks @xmunoz !

**Pull request merged**